logic/src/commonMain/kotlin/com/wire/kalium/logic/data/connection/ConnectionRepository.kt

```diff
@@ -226,7 +227,7 @@ internal class ConnectionDataSource(
                         teamId = null,
                         protocolInfo = ConversationEntity.ProtocolInfo.Proteus,
                         creatorId = connection.from,
-                        lastNotificationDate = null,
+                        lastNotificationDate = DateTimeUtil.currentIsoDateTimeString(),
```

Comment:
Now I'm not sure, won't that create similar issue as the one Vitor debugged last night? For connections maybe we can use `lastUpdate` field?

Reply:
we don't have access to that field! this is for the conversation that we don't have anything for it and instead of null we set the current time to it.